dokuwiki-text/src/main/java/org/xwiki/contrib/dokuwiki/text/internal/input/DokuWikiInputFilterStream.java

```diff
@@ -282,29 +282,28 @@ private void readDocument(File directory, String dokuwikiDataDirectory, DokuWiki
                     proxyFilter.beginWikiDocument(documentName, FilterEventParameters.EMPTY);
 
                     FilterEventParameters documentLocaleParameters = new FilterEventParameters();
-                    String metadataFileContents = FileUtils.readFileToString(new File(fileMetadataPath), "UTF-8");
-                    MixedArray documentMetadata = Pherialize.unserialize(metadataFileContents).toArray();
-                    readDocumentParametersFromMetadata(documentMetadata, documentLocaleParameters);
 
-                    //Wiki document revision
-
-                    if ((documentMetadata.getArray(KEY_CURRENT).getArray(KEY_DATE).containsKey(KEY_CREATED)
-                            && documentMetadata.getArray(KEY_CURRENT).getArray(KEY_DATE).containsKey(KEY_MODIFIED))
-                            && documentMetadata.getArray(KEY_CURRENT).getArray(KEY_DATE).getLong(KEY_CREATED)
-                            < documentMetadata.getArray(KEY_CURRENT).getArray(KEY_DATE).getLong(KEY_MODIFIED))
-                    {
-                        //Wiki Document Locale
-                        proxyFilter.beginWikiDocumentLocale(Locale.ROOT, documentLocaleParameters);
-                        //read revisions
-                        readPageRevision(file, dokuwikiDataDirectory, proxyFilter);
-                    } else {
-
-                        String pageContents = FileUtils.readFileToString(file, StandardCharsets.UTF_8);
-                        String convertedContent = parseContent(pageContents);
-                        documentLocaleParameters.put(WikiDocumentFilter.PARAMETER_CONTENT, convertedContent);
-                        //Wiki Document Locale
-                        proxyFilter.beginWikiDocumentLocale(Locale.ROOT, documentLocaleParameters);
-                        readAttachments(pageContents, file, dokuwikiDataDirectory, proxyFilter);
+                    try{
+                        String metadataFileContents = FileUtils.readFileToString(new File(fileMetadataPath), "UTF-8");
```

Comment:
If missing metadata file is expected to you not generate an exception in such a cache and check if the file exist before trying to read it.